src/room/participant/LocalParticipant.ts

```diff
@@ -231,24 +232,31 @@ export default class LocalParticipant extends Participant {
 
   /**
    * Publish a new data payload to the room. Data will be forwarded to each
-   * participant in the room
+   * participant in the room if the destination argument is empty
    *
    * @param data Uint8Array of the payload. To send string data, use TextEncoder.encode
    * @param kind whether to send this as reliable or lossy.
    * For data that you need delivery guarantee (such as chat messages), use Reliable.
    * For data that should arrive as quickly as possible, but you are ok with dropped
    * packets, use Lossy.
+   * @param destination the participants who will receive the message
    */
-  publishData(data: Uint8Array, kind: DataPacket_Kind) {
+  publishData(data: Uint8Array, kind: DataPacket_Kind, destination: RemoteParticipant[] = []) {
```

Comment:
It would be more flexibile if it could take in an sid array instead of RemoteParticipant, or perhaps either option.
```suggestion
  publishData(data: Uint8Array, kind: DataPacket_Kind, destination?: RemoteParticipant[] | string[]) {
```